.github/workflows/ci.yml

```
@@ -258,6 +258,31 @@ jobs:
         DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         DOCKERHUB_REPOSITORY: ${{ secrets.IMAGE_NAME }}
 
+  # validate the swagger docs
+  swaggerci:
+    if: github.repository == 'meshery/meshery'
+    name: swagger-docs
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: "master"
+    - name: Check if swagger was modified
+      uses: dorny/paths-filter@v2
+      id: changes
+      with:
+        filters: |
+          modified:
+            - added|modified: 'handlers'
+    - name: Setup go-swagger
+      uses: minchao/setup-go-swagger@v1
+      with:
+        version: v0.26.1
+    - name: swagger-spec
+      run: swagger generate spec -o ./helpers/swagger.yaml --scan-models
```

```suggestion
      if: steps.filter.outputs.modified == 'true'
      run: swagger generate spec -o ./helpers/swagger.yaml --scan-models
```
I can't add this here without weird html tags lol. Anyways its just placing the if condition here as well.